feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.component.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  OnDestroy,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Observable, Subscription, timer } from 'rxjs';
+import { debounce, distinct, take } from 'rxjs/operators';
+import { ConfiguratorUISettings } from '../../config/configurator-ui-settings';
+
+export interface Quantity {
+  quantity: number;
+}
+
+export interface ConfiguratorAttributeQuantityComponentOptions {
+  allowZero?: boolean;
+  initialQuantity?: Quantity;
+  disableQuantityActions?: Observable<boolean>;
```

Comment:
can we use our convention to append `$` to Observable variable name as `disableQuantityActions$`

Reply:
pick